spec/amp-email-format.md

```diff
@@ -0,0 +1,379 @@
+
+* Does this mean email senders can track when I interact with a carousel?
+
+    * Request for images in the carousel can indicate to the sender that the user is interacting with the message.
+
+* What will happen if I open an AMPHTML Email in an email client without AMPHTML support?
+
+    * The usual HTML content will be rendered. Email clients currently ignore MIME types they don’t understand. For example the [`text/watch-html` MIME type](https://litmus.com/blog/how-to-send-hidden-version-email-apple-watch) gets ignored according to the rules specified in [RFC 1341 7.2.3](https://www.w3.org/Protocols/rfc1341/7_2_Multipart.html) which state:<br><br>*“In general, user agents that compose multipart/alternative entities should place the body parts in increasing order of preference, that is, with the preferred format last. For fancy text, the sending user agent should put the plainest format first and the richest format last. Receiving user agents should pick and display the last format they are capable of displaying.”*
+
+* Will users be able to opt out?
+
+    * Email clients may offer users the ability to disable AMPHTML documents in their email.
+
+* Who else is supporting rendering AMPHTML emails?
+
+    * Currently, Gmail is the first adopter with their [Gmail Developer Preview](https://gsuite.google.com/campaigns/index__amp-for-email.html)  starting with the Web developer preview. We're looking forward to working with other mail providers and clients to have their web and mobile apps support this as well.
+
+* Will this consume more battery life?
+
+    * That will depend on the implementation strategy chosen by the email clients.
+
+* Will this enable people to send me Bitcoin miners in my email?
+
+    * No. Valid AMP documents are not permitted to have their own Javascript.
+
+* How will AMPHTML Email deal with spam since attackers can evade spam detection in real time at mail-open time?
+
+    * Email providers will need to proxy all XHRs and use the responses to render the content and perform phishing checks.
+
+* Are there any CSS restrictions?
+
+    * Yes. For a reference list of CSS properties and values that are allowed within email messages, see [Gmail Supported CSS Properties & Media Queries](https://developers.google.com/gmail/design/reference/supported_css). Furthermore, the [AMP Validator](https://github.com/ampproject/amphtml/tree/master/validator) can be used to check them or [AMP by Example’s Playground](https://ampbyexample.com/playground/#runtime=amp4email) with the "AMP for Email" runtime selected.
+
+* Will the AMP `CLIENT_ID` be supported?
+
+    * No.
+
+* Will [AMP Variable Substitution](https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md) be supported?
+
+    * No. They are mainly used in `amp-analytics` and `amp-pixel`, that are not supported in AMPHTML Email. In addition, page URLs used for variable substitution do not apply to emails.
```

Comment:
I believe we currently allow only the `RANDOM` variable. @raywainman

Reply:
I don't thin is supported, mutation of URLs in amp-list are disallowed and I am not sure it makes sense for form inputs.